docs/server/source/production-deployment-template-tendermint/workflow.rst

```diff
@@ -0,0 +1,138 @@
+Overview
+========
+
+This page summarizes the steps *we* go through
+to set up a production BigchainDB + Tendermint cluster.
+We are constantly improving them.
+You can modify them to suit your needs.
+
+.. Note::
+    With our BigchainDB + Tendermint deployment model, we use standalone MongoDB
+    (without Replica Set), BFT replication is handled by Tendermint.
+
+
+1. Set Up a Self-Signed Certificate Authority
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We use SSL/TLS and self-signed certificates
+for MongoDB authentication (and message encryption).
+The certificates are signed by the organization managing the cluster.
+If your organization already has a process
+for signing certificates
+(i.e. an internal self-signed certificate authority [CA]),
+then you can skip this step.
+Otherwise, your organization must
+:ref:`set up its own self-signed certificate authority <how-to-set-up-a-self-signed-certificate-authority>`.
+
+
+.. _register-a-domain-and-get-an-ssl-certificate-for-it-tmt:
+
+2. Register a Domain and Get an SSL Certificate for It
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The BigchainDB APIs (HTTP API and WebSocket API) should be served using TLS,
+so the organization running the cluster
+should choose an FQDN for their API (e.g. api.organization-x.com),
+register the domain name,
+and buy an SSL/TLS certificate for the FQDN.
+
+.. _things-each-node-operator-must-do-tmt:
+
+Things Each Node Operator Must Do
+---------------------------------
+
```

Comment:
There should be a line here that starts with an open checkbox ☐ saying what the node operator should do. Are they supposed to choose names for all these things? Should those names be unique within the BigchainDB network?

Reply:
Yes, they should be, I will update.

Reply:
:+1: